crypto/err/err_prn.c

```diff
@@ -36,7 +36,8 @@ void ERR_print_errors_cb(int (*cb) (const char *str, size_t len, void *u),
             data = "";
         hex = OPENSSL_buf2hexstr((const unsigned char *)&tid, sizeof(tid));
         BIO_snprintf(buf, sizeof(buf), "%s:error:%s:%s:%s:%s:%d:%s\n",
-                     hex, lib, func, reason, file, line, data);
+                     hex == NULL ? "" : hex, lib, func, reason, file, line,
```

Comment:
Same here.

_(If you prefer to check `hex == NULL` instead of `hex != NULL` as I did above, that's ok for me too._)